ci/scripts/collect-sources-analysis.sh

```diff
@@ -3,8 +3,6 @@
 set -e
 
 #Cmake
-rm -rf /src/build
```

Comment:
It was observed that files were kept from one build to another

Reply:
no longer needed